### DISCUSSION TITLE
ModuleNotFoundError: No module named 'altair'

### DISCUSSION THREAD

**Discussion by Cylis78** [azuritecoin/OnnxDiffusersUI#55] [Q&A]
When I run the 'pyhton onnxUI.py' command, I keep getting the above error, and it never gets to the "Running on local URL:" step.

Any help in resolving this would be greatly appreciated.

UPDATE: I'm an idiot, and new to python...
All I had to do was run 'pip install altair vega_datasets' ... Incase any other newbies have the same issue

**Comment by azuritecoin** [comment]
Looks like the gradio package updated yesterday and didn't add altair to their requirements list

**Comment by TuneBG** [comment]
The thing i did is to remove the line """import altair as alt""" 
which is in components.py
located in \virtualenv\Lib\site-packages\gradio\

and now it runs no problem

**Comment by azuritecoin** [comment]
it looks like they fixed the bug in the latest release of gradio: https://github.com/gradio-app/gradio/releases/tag/v3.13.0b1
